internal/proto/encryptionkeyrotation.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+package encryptionkeyrotation;
+
+option go_package = "github.com/csi-addons/kubernetes-csi-addons/internal/proto";
+
+// EncKeyRotation holds the RPC method for running
+// key rotation operation on the volume.
+service EncKeyRotation {
+  rpc EncKeyRotate(EncKeyRotateRequest)
```

Comment:
as its a proto file lets not use short form. use `Encryption` instead of `Enc` in all other places in proto.

Reply:
I did that to avoid the proto namespace collision. It cannot be Encryption else it will conflict with the ns in spec.

Any other alternatives?

Reply:
can you please explain a bit on conflict with the NS, i might be missing something.

Reply:
If I name the service `EncryptionKeyRotation` it will be the same as the service name in the imported spec from `cis-addons/spec`. It will compile fine but will panic at the run time due to confilicting namespaces, i.e. both of them (`internal/proto` and `csi-addons/spec`) having the same member names `EncryptionKeyRotation`.

Such collision is not allowed in protobuf.

Reply:
EncryptionKeyRotationController is the serviceName in the csi-addons/spec and here it will be EncryptionKeyRotation, isn't it?

Reply:
The above statement was just an example. This applies to message names as well.

`EncryptionKeyRotateRequest` and `EncryptionKeyRotateResponse` are the conflicting members.